sandbox/index.js

```diff
@@ -1,11 +1,47 @@
 import { addTranslations, setLanguage } from '../src/lib/i18n.js';
 import { updateRootContext } from '../src/lib/smart-manager.js';
-import { lang, translations } from '../src/translations/translations.en.js';
+import { lang as langEn, translations as translationsEn } from '../src/translations/translations.en.js';
+import { lang as langFr, translations as translationsFr } from '../src/translations/translations.fr.js';
 import '../src/components/cc-toaster/cc-toaster.js';
 import '../src/components/cc-notice/cc-notice.js';
+import '../src/components/cc-toggle/cc-toggle.js';
+
+console.log(langEn);
+console.log(langFr);
+
+addTranslations(langEn, translationsEn);
+addTranslations(langFr, translationsFr);
+
+const DEFAULT_LANG = langEn;
+let currentLang = localStorage.getItem('cc-component-lang');
+if (currentLang == null) {
+  currentLang = DEFAULT_LANG;
+  localStorage.setItem('cc-component-lang', currentLang);
+}
+setLanguage(currentLang);
+
+const $langContainer = document.querySelector('.languages');
+
+const languageToggle = document.createElement('cc-toggle');
+languageToggle.choices = [
+  {
+    label: `🇬🇧 ${translationsEn.LANGUAGE}`,
+    value: langEn,
+  },
+  {
+    label: `🇫🇷 ${translationsFr.LANGUAGE}`,
+    value: langFr,
+  },
```

Comment:
nitpick: country flags should never be used for language pickers :wink: 

https://wplang.org/never-use-flags-language-selection/